apps/web/src/services/api/page-service.ts

```
@@ -570,8 +578,10 @@ export const pageService = {
           context: mutationContext,
           tx,
         });
+        if (result.deferredTrigger) deferredTriggers.push(result.deferredTrigger);
       }
     });
+    deferredTriggers.forEach(t => t());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)